test/regression_tests.sh

```diff
@@ -15,9 +15,17 @@ TEST_EXECUTION_LOG_FILE="/tmp/test_exec.log"
 install_torchserve_from_source() {
   echo "Cloning & Building Torchserve Repo from " $1
 
+  #uninstalling existing  torchserve torch-model-archiver if any
+  pip uninstall --yes torchserve
+  pip uninstall --yes torch-model-archiver
+
   # Install dependencies & test dependencies
-  pip install torch torchtext torchvision sentencepiece psutil future
-  sudo apt-get install npm
+  pip install  torch torchtext torchvision sentencepiece psutil future
+  pip install  mock pytest pylint pytest-mock pytest-cov
+  apt-get -y install nodejs-dev node-gyp libssl1.0-dev npm
+  npm install -g n
+  n latest
+  export PATH="$PATH"
   npm install -g newman newman-reporter-html
 
   # Clone & Build TorchServe
```

Comment:
@shivamshriwas @mycpuorg We should use the new install from src script instead as part of PR# https://github.com/pytorch/serve/pull/322

Reply:
Done

Reply:
We have added test cases for all examples in model zoo which cover both default and custom handler (eg resnet152 and mnist model).

added https test cases covering
- get inference/management api description
- register model
- scale model sync/async
- unregister model

Reply:
@chauhang Can you confirm if all the changes that you had requested have been taken care of?